server/src/api/v1/orders/get-orders.ts

```diff
@@ -65,6 +66,30 @@ export async function getOrders(req: Request, res: Response) {
     customResolver.set('is_active', (field: string, req: Request, value: string) => {
         return true;
     });
+    if (!Vars.currentUser.is_admin) {
+        let result: Customer[] = await Customer.findAll(
+            {
+                attributes: ['id'],
+                where: {
+                    userId: Vars.currentUser.id
+                },
+                raw: true
+            })
+            .catch(error => {
+                success = false;
+                return [];
+            });
+        if (!success) {
+            return res.status(500).send(wrapResponse(false, { error: 'Database error' }));
+        }
+        let customerIds: string[] = [];
+        result.forEach(el => {
+            customerIds.push(el.id);
+        });
```

Comment:
```suggestion
        let customerIds: string[] = result.map(el => { return el.id; });
```

Als Tipp 
https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
Code oben bitte vorher testen, war jetzt nur so hingeschrieben :)

Reply:
siehe neuster commit